compiler/circle-execution-plan/src/ExecutionPlanner.h

```diff
@@ -20,6 +20,10 @@
 #include "TargetPlatform.h"
 #include <luci/IR/Module.h>
 #include <luci/Plan/CircleNodeExecutionPlan.h>
+#include "IScratchpadHelper.h"
+#include "ScratchpadHelperLinux.h"
+#include "ScratchpadHelperMCU.h"
+#include "ScratchpadHelperCMSISNN.h"
```

Comment:
for reviewers:
I tried to place these includes 
```c++
#include "IScratchpadHelper.h"
#include "ScratchpadHelperLinux.h"
#include "ScratchpadHelperMCU.h"
#include "ScratchpadHelperCMSISNN.h"
```
 higher than includes with `<...>`, but in this case, the compiler stops seeing what is (for example) `uint32_t` or `luci::CircleConv2D` and smth else.
I found that this is the most simple variant for me to solve this problem, but if there are another solutions, I will be glad to add them.

Reply:
This is exactly why we try to place headers as high as possible =)
This error means that `IScratchpadHelper` and related headers are not "self-sufficient", i.e. they use something that is not defined/included in them.

For this particular error with `uint32_t` you need to add `#include <cstdint>` in `IScratchpadHelper.h`

I'll check if there are other problems in few minutes